README.md

```diff
@@ -1,56 +1,128 @@
-* [Contribute to Open Liberty](https://github.com/OpenLiberty/open-liberty#contribute-to-open-liberty)
-* [Community](https://github.com/OpenLiberty/open-liberty#community)
+* The Open Liberty modular architecture means only the features you need are loaded at runtime to help reduce image size and memory consumption. The smaller footprint means you can run more application instances per machine to help reduce production costs.
 
-## Getting Started
+* Servers running Open Liberty start faster, making them ideal for rapid scale-up and scale-down of containerized applications.
 
-### Downloading
-You can download released versions and nightly build artifacts from: https://www.openliberty.io/downloads/
+## Getting Started
 
-### Quick start
+### Launch a sample app using [Maven](https://maven.apache.org/)
     git clone https://github.com/OpenLiberty/sample-getting-started.git
     cd sample-getting-started
     mvn clean package liberty:run-server
 
 Open browser to http://localhost:9080/ and explore the demo application.
 
-### Open Liberty Guides
-Visit the [OpenLiberty website](https://openliberty.io/guides/) for a number of step by step guides.
+This [guide](https://openliberty.io/guides/getting-started.html) shows how it was built
+
+### Prerequisites
+
+* Java SE 8,11,12 - [More details](https://openliberty.io/docs/ref/general/#java-se.html)
+
+### Downloads
+
+* Zipped released versions and nightly build artifacts from: https://www.openliberty.io/downloads/
+* Official Released Open Liberty Docker containers from : https://hub.docker.com/_/open-liberty
+
+## Usage
+
+### Controlling Open Liberty
+
+The bin directory contains a server script to help control the server process.
+The script supports the following actions:
+
+Command| Does...
+:-----:|:-----:
+`create`|creates a new server
+`start`|launches the server as a background process
+`run`|launches the server in the foreground
+`debug`|launches the server in the foreground with JVM debug options
+`stop`|stops a running server
+`status`|check to see if a specified server is running
+`package`|packages server runtime and target server configuration/application(s) into zip archive file
+`dump`|dump diagnostic information from the server into an archive
+`javadump`|dump diagnostic information from the server JVM
+`list`|list existing servers
+`version`|displays the version of the server runtime
+`pause`|pause all the components in the server that can be paused
+`resume`|resume all paused components in the server
+`help`|get command-line/script help, including descriptions of additional options
+
+To use the script on Windows (the .bat extension is optional):
 
+    bin\server.bat create <serverName>
+    bin\server.bat start <serverName>
+    bin\server.bat help
 
-## Contribute to Open Liberty
+To use the script on other platforms:
+
+    bin/server create <serverName>
+    bin/server start <serverName>
+    bin/server help
+
+
+### Open Liberty Docs
```

Comment:
I don't think this section is necessary. We already have it linked in the repo description and at the top of the readme with a badge icon, and by clicking on the big openliberty logo icon